salon_creator/lib/screens/login_page.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:salon_creator/authentication/sign_in.dart';
+
+class LoginPage extends StatefulWidget {
+  const LoginPage({Key key}) : super(key: key);
+
+  @override
+  _LoginPageState createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(children: [
+        new Container(
+          decoration: new BoxDecoration(
+            image: new DecorationImage(
+              image: new AssetImage("assets/background.jpg"),
+              fit: BoxFit.fill,
+            ),
+          ),
+        ),
+        Container(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Center(
+                child: Container(
+                  width: 300,
```

Comment:
This might behave weirdly when viewed on different screen sizes (iPhone vs iPad). Maybe use media query to make this a proportion of the screen width? Also, it might be worthwhile to test how the screen looks on an iPad in the simulator.
https://stackoverflow.com/a/43130622